Playbooks/Create Incidents with Email/Readme.md

```diff
@@ -0,0 +1,46 @@
+
+
```

Comment:
Readme file will look good, if you can mention and explain, the use cases where this playbook can be utilized in infra or SOC , SOAR , if possible

Reply:
added.